content/learn/node-file-descriptors.en.md

```diff
@@ -57,7 +57,7 @@ async function example() {
     console.log(filehandle.fd);
     console.log(await filehandle.readFile({ encoding: 'utf8' }));
   } finally {
-    await filehandle.close();
+    await filehandle?.close();
```

Comment:
Instead of optional chaining could we have an `if` statement?

Some people reading this content might just not know about what optional chaining is.

Reply:
Yes, of course.